### DISCUSSION TITLE
On the falsifiability of the Dark Sector (LCDM and extensions) and the revolution that awaits us this december

### DISCUSSION THREAD

**Discussion by LifeIsStrange** [a-cosmology-group/acg#243] [Astrophysics and Physics]
I believe it was @Budrap that answered me about how much Dark Matter and Dark energy are anti-science, non falsifiable fudge factors, reminiscent of the medieval "Dark Age" and of past hypothetical concepts like the Vulcan planet(oids) or aether theories (albeit the latter are still relevant). 
I never answered such comment(s) (sry didn't find the time/energy) so I thought I could talk a bit about that topic.

There are btw many current "Dark" objects/concepts both in astrophysics and in cosmology. E.g. another missing planet as a candidate explanation for the major problem of the Kuiper belt cliff in our solar system. Many extensions like quintessence are "dark" and I'd argue the classification or wether a theory/concept is Dark and how much on a continuum it is "Dark" (aka non observable or non interacting) is not clear. IMO many concepts in alternative cosmology can have a comparable level of darkness, actually is is almost an intrinsic property for an idea to be dark, assuming it is not yet proven to exist given current observations. Any truly "new physics" is to a large extent dark because of the truism that we have sky surveys of all type of objects and of all kinds of radiation frequencies (panchromatic).
It is true that any law or matter that is only observable through (anti)gravity is amongst the darkest possible theories but are IMO close to dark or gray theories that only interact with gravity and with the weak force (such as neutrinos extensions which are most likely required in a universe in equilibrium).
A few years ago I actually had the same opinion as Budrap as I thought those were fudge factors that were unfalsifiable and maximally unparcimonous epistemologically (Ockham razor, etc, e.g. because of the quantity of dark matter required versus baryonic matter)
Of course a theory can be evaluated/compared according to its intrinsic complexity, degree of agreement with data, predictions, expressive power (a theory that predicts everything predicts nothing) and tunability, and the degree of finetuning/coincidences.

But actually, nowadays, I find the LCDM cosmology and many of its extensions, very testable/falsifiable empirically,
moreover I'd argue that static universes require a dark energy (cosmological constant or not) in order to stabilize gravity.
Moreover, there are much more possible configurations of universes that drift (contract or expand) than ones that are truly stable cf (schrodinger paper about einstein static universe). Contrary to popular belief there can be stable static universes (e.g. via some topologies, or via dynamic dark energy or via some modified theories of gravity or via Einstein-cartan/teleparallel gravity).
So while the amount of dark energy needed in (most but not all) static universes is different (what should the value be?) it is IMHO a double standard and hypocrisy to criticize blindly dark energy while ignoring the fact it most likely is needed in other cosmologies.
And empirical evidence recently shows that most likely dark energy observations are plain wrong and an artifact of supernovae dust reddening, if so that would be an even bigger loss for static universes than for an expanding one (not sure about that but IMO there is the ironically missed argument, that dark energy in static universes is probably much darker than dark energy in LCDM).
Also the difference epistemologically between a new particle versus a new or modified energy/field/wave is not as much more parcimonous as I once thought (e.g. dark matter vs MOND).

I shall do a proper state of the art review of the various cosmological tests/probes latter but quickly:
There is the Tolman test and the distance duality test, that empirically test the necessary (non fudgeable) divergence between both the luminosity distance and the angular distance in a static universe versus an expanding one. So this allows to completely disentangle those cosmologies that otherwise mostly predicts the same redshift distance.
Even the redshift distance can diverge as show recent Hubble diagrams measurements.
There is also cosmic time dilation, the Alcock Pazynsky test with BAO, the various tests of evolutions (are merger, shape, number density, SFR, metallicity, etc constant or evolving over redshift), the impossibly early problem, reproducing the properties of the "CMB", the chemical abundances (BBN, lithium problem), of cosmic rays, the rotation curve of galaxies, the redshift drift, the faint young sun, void cosmology and many other empirical cosmological tests/probes.

One might argue that those tests/probes shows that an expanding universe is well falsifiable but that the dark components (matter and energy) remains non falsifiable. Well one, if we refute expansion, we mostly but not completely refute the dark components so it is transitive, secondly those tests highly constrain the dark components and their extensions (e.g. the second peak of the CMB).

Moreover I'd argue the Dark sector (real term for listing dark concepts like DM and DE) can be and will be along with most of their extensions, largely falsified (or.. proven) within the next 1-3 years.
Which is remarkable, revolutionnary and excellent science so if the Dark sector of the standard model was for a long time, low falsifiable and effectively behaved as forever tunable invisible fudge factors, it appears they are as or even much more falsifiable than many core ideas in alternative cosmology. (even though those next 3 years will probably be a bloodbath for alternative cosmologies too.
People don't understand cosmology is the science that will be the most revolutionized of all, of the century, within said 1-3 years.
This is because we are entering the Decade of the surveys (I will make a post about them).
One might wonder, why now? Why not within the last 24 years? Why not in the distant future?
There are multiple reasons to this:
One might believe it's because of SpaceX and the significant reduction in the cost of launching satellites/telescopes in space.
I don't believe this at all, sending telescopes in space has always been much cheaper than building said telescopes/instruments.
In many regards, mankind has actually regressed,
1) telescopes are no longer serviceable (hubble was repaired in space, and its camera was updated too via the defunct space shuttles)
albeit this is coming back thanks to China that will shows the obvious way, by making its Xuntian telescope periodically dock their space station for free.
2) Ground based optical mirror size has peaked in the 90s.
This will radically change with the E-ELT and the cherenkov telescopes (CTA). But this isn't needed for the inpending revolution.
3) The obvious sweet spot is to put a large telescope in an airplane (like the defunct antonov) to mostly bypass the atmosphere for cheap.
The only implementation is retired. Trivial concept was never scaled.
4) Balloon astronomy mostly died for a decade
It is currently being resurrected but again isn't needed for the revolution
5) Both the ISS and hubble will soon be retired
6) The most major extreme regression is the purist and inept trend of sending telescopes in lagrangians, which force them to consume fuel instead of being e.g. geosynchronous like Hubble, meaning instead of living 2 or 4 decades like many space telescopes (or even in some cases, possibly a century worth of observations), those telescopes have pathetically low lifespans (usually ~5 years).
L2 might have a bit less noise but this effect, except maybe for niche spectra like far infrared, is IMO considerably inferior to the added scientific values of multiplying the science data quantity (and transmission rate) by 4 times or more.
6) There is this fundamental concept in sociology or in irationalitiology of Artificial Scarcity, meaning an effective "poverty/weakness/stagnation" that is contingent, artificial, a product of manking mediocrity and not of real costs or difficulties.
I believe there is widespread scientific fraud in NASA missions that goes hyper overbudget and with extreme delays.
The prime example is JWST which people erroneously retrospectively see as a huge success (which it is _relative_ to what we have..)
it is called "The telescope that ate astronomy", with a budget worth the new LHC particle accelerator (iirc 20 billions).
https://www.nature.com/articles/4671028a
Launching a large payload in space cost ~50 millions or 1/400th of the total cost.
We still, 34 years after the launch of Hubble, not launched a single optical telescope with a larger mirror (despite hubble mirror being quite small and smaller than originally planned).
And guess what, all incoming space telescopes are smaller than Hubble! (euclid, Xuntian, Roman).
The only other telescope larger than Hubble was hershel, which was IMO an extreme failure with dubious scientific value (compared to the insane scientific value of a larger optical or UVOIR telescope).
People don't know how shameful and despicable it is that the USA have sent over ~30 Hubble class space telescopes in the meantime, but they are simply pointing at earth for spying. They have actually sent mirrors larger than Hubble one.
And they are the reason why both Hubble and Roman mirror size is so low.
We will have to wait (unless China wakes up NASA/ESA fraud and complacency) for LUVOIR in 2039 (+ delays).
In other terms, astronomy, cosmology and astrophysics have to wait 50 fucking years for launching a dumb telescope in space larger than 2 fucking meters. It is insane mediocrity and a crime against humanity.
The cost of such a geosynchronous satellite is under 500 million dollars and it is the absolute bottleneck of the three most fundamental sciences.
Note that there is a scenario where chineses incrementally increase the size of the Xuntian within their space station by shipping and plugging segmented mirrors one at a time but I don't believe they would try such a promising idea.
Because of this many cosmological probes will be bottlenecked, because they are resolution/redshift limited (diverge more with increasing redshift).
Hubble despite having an almost three time smaller aperture than JWST, was able to detect a galaxy at Z=11, the reason is because the resolution of a telescope increase with the wavelength frequency (IIRC optical is 10 time the resolution of mid infrared).
As such it is a crime and a sad joke that JWST with its 20 billion budget was not able to ship with a 50$ optical camera, as it would have allowed to detect far photometric redshifts far higher than JWST ~z=20 limit and it would have allowed much better morphological analysises, which would have probably 100% falsified LCDM. Indeed near infrared is the sweet spot for spectroscopic redshift and is therefore complementary and a false dilemna, e.g. Xuntian allows simultaneous observations in UV, Optical and near infrared.
Most importantly the biggest missed science opportunity of the century was that JWST cameras are extremely obscolete (narrow)
It is not hard to have a wide field near infrared camera as shows euclid, xuntian and roman, but it would have been absolutely trivial to put a wide field optical camera, the technology being mature pre JWST launch.

Let's move on from this rant/digression about missed opportunities (which are IMO crimes against humanity)
So why now? Why will cosmology be revolutionized in the next 3 years?
As said, it mostly won't come from those 6 listed points.
There is the E-ELT which a large aperture.
Wiki says:
> It has around 250 times the light gathering area of the [Hubble Space Telescope](https://en.wikipedia.org/wiki/Hubble_Space_Telescope) and, according to the ELT's specifications, would provide images 16 times sharper than those from Hubble.[[4]](https://en.wikipedia.org/wiki/Extremely_Large_Telescope#cite_note-E-ELT-4)

I don't believe this at all. The scaling resoluton of ground based telescopes seems to me, to be mostly capped at ~3 meters. I don't see an appreciable difference in resolution between 3 meters ground based telescopes and the 10.4 meters GTC telescope.
IMO they are mostly bottlenecked by the atmosphere and progress in https://en.wikipedia.org/wiki/Adaptive_optics has stagnated.
So unless the E-ELT revolutionize adaptive optics (wouldn't it be easier to do on smaller telescopes?) I believe it will be an immense failure (albeit their budget is surprisingly reasonable). I hope to be wrong on that one as it would allow e.g. to measure the redshift drift and to falsify the age of the universe.

So the revolutions comes from other things that max redshift/optical resolution.
One: we have an absolute revolution in wideness.
Euclid camera has ~100 times the wideness of Hubble WFC3 and Xuntian/LSST ~500 times.
LSST effectively has the largest camera ever built and xuntian is close and will be spaceborn.
Therefore they should be close (albeit a bit inferior) to Hubble in resolution but cover a surface that is 500 time wider.
This means that 3 years of LSST is equivalent to 1500 years of Hubble observations.
(I'm not factoring in other aspects of the camera that might be improved, like the FPS/latency?) plz answer on that one
3 years - 1500 years.
Just for LSST, which will happen in paralllel to euclid, xuntian, roman and E-ELT.
This level of progress in terms of data acquisition is like teleporting in the future, it is a sci-fi level advancement for our civilization.
And it will not only happen in optics but also in UV and in near infrared, in parallel.
Euclid first data release **this december** will be close to 100 years of continuous Hubble observations, and the world will never be the same.
Moreover the revolution is completely panchromatic, erosita is already released and allows to scan 50% of the xray universe
the CTA improve bounds on gamma rays by order of magnitudes
FAST in 2 years is equivalent to 13 years of pulsar time observations by NANOGRAV, hence next year it will double our constraints on gravitational waves/background.
SKA will revolutionize radio astronomy again by order of magnitudes
Xuntian, CASTOR, sepktr UV and ULTRASAT will revolutionize UV astronomy (and therefore e.g. observations on GRBs and the first days of supernovaes) (and little known, will also increase photometric redshift accuracy)
near infrared and optical cf above + others like spherex
PLATO will revolutionize exoplanet search and asteroseismology
astrometry via GAIA DR4 and radial velocities
The only band that is leftover is the far infrared (that is too hard to observe anyway)

There is also two other revolutions in parallel of the wide cameras and of complete panchromatism
All those observations will considerably increase the opportunities for spectroscopic confirmation and analysises.
Thankfully spectrographs "fiber robots" have also simultaneously revolutionized, with studies like DESI, 4MOST, WEAVES, etc spectroscopy both extragalactic and of the milky way will be increased by multiple order of magnitudes (revolutionizing chemical abundances, stellar models and radial velocities)
The third revolution is the time domain/transient astronomy, as those instruments and some dedicated ones like ULTRASAT and spherex will scan the whole universe with a timing resolution improved by multiple order of magnitudes. This should extremely highly constrain the dynamic universe properties and reveal the degree of homeostasis. Moreover it will revolutionize GRB, supernovae, FRB and time delay astronomy.
And while deepness will not be better than Hubble for UVOIR, other bands will have strong increase in resolution and the mean deepness in UVOIR will considerably increase, via e.g. middle grounds like the PAU survey (that btw reinvent photometric redshift by making them ~as accurate as spectroscopic).

So it is basically almost every single aspects of astronomy that are simultaneously revolutionized in the next 1-5 years (most to come in <3 years), and this mindblowing news will indeed allow extreme synergies in cosmological constraints.

Additionally CMB (and CRB and to a lower extent neutrinos) observations will too be revolutionized.
Few people know that the CMB measurements of spectral distortions have not improved since 45 years (COBE).
As WMAP and Planck only improved anisotropies and polarimetry observations (spectrometer vs bolometer).
Yet the strongest refutation of the CMB as being primordial comes from the absence of dust spectral distortions, both y and u distortions must (guaranteed by theory) be observed in the next few years (e.g. via BISOU) (sadly pixie isn't financed but won't be needed). This will give a _definitive_ refutation of the primordial CMB.
In parallel and synergetically, there will be the BICEP3, Simons, CMBS4 and litebird surveys that will give exquisitely extreme constraints on the CMB (will probably allow to detect wether it is in the solar system), which will obviously (mark my words) makes the current Hubble tension seems benign in comparison.
They (Simons being first probably) will officially and definitively falsify standard inflation (slow roll inflation) via δr.
It will also gives a definitive CMB POV of the curvature of the universe (which is currently in tension)
The spectroscopic and astrometric revolution will gives a definite answer to the mass and rotation curve of the milky way, moreover GAIA will give a definitive measurement of the non newtonian gravity acceleration in wide binaries and the countless gravitational lenses detected by euclid, will allow to proves defintiely that rotation curves are infinitely flat (successor of https://tritonstation.com/2024/06/18/rotation-curves-still-flat-after-a-million-light-years/ )
So we will have 3, direct and distinct (among many others) and definitive falsficiations of dark matter (and of most MONDs)
UV and time domain astronomy should give a definitive answer to dark energy (cf latest DES study showing negative dark energy AKA a purely dust based artifact with no physical reality).

So yes LCDM is poor science made of fudge factors but for dark components, they are actually much brighter than given credit for.
In 3 years, everything, wether the universe is expanding, wether dark matter exist, dark energy and inflation (at least the reasonable models of inflation) will be ALL empirically falsified simultaneously via distincts independent probes with measures far above 5 sigmas.
Most of that actually will already happen **this december** with 100 years worth of Hubble time.
And it doesn't stop there,
We will have definitive constraints on neutrino mass and behaviours, on MACHOs, on modified gravity, on the nature of the CMB, on the homeostasis of the universe, on varying mass electrons/photons, on varying speed of light, on Alcock paczynski, etc
Moreover, extensions of dark matter and dark energy will be almost all ruled out or made negligible.
As shows for example: https://arxiv.org/abs/2406.18274
There will be also a birth of many new cosmological tests/probe that were unfeasable previously (eventually up to the redshift drift)
such as https://arxiv.org/abs/2406.17849
https://arxiv.org/abs/2406.18298
https://arxiv.org/abs/2307.02117
etc..

**Comment by HanDeBruijn** [comment]
>moreover I'd argue that static universes require a dark energy (cosmological constant or not) in order to stabilize gravity.

Nope. With the [Variable Mass Theory](https://www.youtube.com/watch?v=T7jVviVBzKs), gravity is stabilized in a very natural way. See [Seeliger's Paradox](http://www.alternatievewiskunde.nl/cosmology/india.htm) at my website.

>Moreover, there are much more possible configurations of universes that drift.

Nature doesn't care about "much more possible configurations", as being made up by arrogant twits. Quoting from one of our pages titled [Euclidean Universe](http://www.alternatievewiskunde.nl/cosmology/olbers.htm) first line:
_Unified Alternative Cosmology (UAC) starts with the assumption that the universe is infinite in space and time. Because of Russel's Paradox and Infinitum Actu Non Datur this effectively means that "**There is no universe**" at all; in the sense that there would exist something that contains everything. Our infinite universe, according to Gauss' dictum, is just a way of speaking._

>to detect far photometric redshifts far higher than JWST $\sim z=20$ limit

According to another [UAC](http://www.alternatievewiskunde.nl/cosmology/helaas.htm#rood) webpage, the maximum possible observable redshift is $z_{max} \approx 28.5$. If that prediction is true, then we have little reason to complain about nowadays equipment available for astronomy. Besides this, has mankind nothing else to worry about?

**Reply by LifeIsStrange** [reply]
I believe your work is very interesting @HanDeBruijn but to be clear your model assume that the photon size/mass (or electron size?) continuously and eternally grow with time.
So I wouldn't consider this theory to be in equilibrium. E.g the size of the electron will become unbalanced versus other particles masses (or is the theory increasing all particles masses while keeping the current ratios?)
At some point a single electron will have the mass of a black hole, etc
Maybe I misunderstand something, not sure.

**Reply by HanDeBruijn** [reply]
@LifeIsStrange .
>your model assume that the photon size/mass (or electron size?) continuously and eternally grow with time.

Mass, no size and only particles with non-zero rest mass.
>is the theory increasing all particles masses while keeping the current ratios?

Yes.
>At some point a single electron will have the mass of a black hole, etc

Good point!

**Reply by HanDeBruijn** [reply]
@LifeIsStrange 
>Good point!

There is a [relevant reply](https://github.com/orgs/a-cosmology-group/discussions/200#discussioncomment-9205176) in thread 200. The black hole state for an elementary particle like the electron is reached for a mass becoming equal to the Planck mass:

$$
m_P=\sqrt{\hbar c/G} \approx 0.2176483258 \times 10^{-7}\mbox{ }kg
$$

It seems that could be the **end** of our increasing Variable Mass (Theory) indeed.

If you go to the webpage titled [Narlikar's Law](http://www.alternatievewiskunde.nl/cosmology/narlikar.htm) and search for "It's typical" then you will find a pointer to another Variable Mass theory by the same professor Narlikar. That other theory does not end with maximal mass but it **starts** with particles having the Planck mass instead of zero mass. Isn't that typical? In the book [_Seeing Red_](http://www.alternatievewiskunde.nl/cosmology/ACG_refs/968368905_Arp.pdf) by Halton Arp both theories are (more or less) discussed: search for "Planck particle".

Whatever. I do not think that increasing mass ends into [Micro black hole](https://en.wikipedia.org/wiki/Micro_black_hole)s. The reason is that the _Hubble parameter_ changes with time. According to (my version of) the Variable Mass Theory it _is decreasing with time_, because $H$ is inversely proportional to an Age (of the universe). The decreasing part BTW also seems to be the mainstream view. Another Google search says that _the Hubble "constant" is a constant only in space, not in time_. Which is also nice because the math of our VMT needs the constant in space part (with distance $r$ ) most of the time. Now let's take a look at the fundamental equation for increasing mass according to the VMT, in (atomic) time domain.

$$
\Large m = m_0.e^{H.t}
$$

And assume a zero timestamp $t=0$ extremely far into the future. Then $H\sim 1/\mbox{Age}$ will be much smaller than today and so the exponential growth of mass will be much less than today. So I'm pretty confident that electrons will not become black holes.

**Comment by LifeIsStrange** [comment]
NASA pioneers program is kinda cool
https://en.wikipedia.org/wiki/Pioneers_Program
especially Aspera in EUV
https://ui.adsabs.harvard.edu/abs/2021SPIE11819E..03C/abstract
https://arxiv.org/abs/2407.15391

> Owing to the spectrum of ˘Cerenkov radiation, most
of the incipient relativistic electrons undergo inverse-Compton scattering with the cosmic microwave background. It
turns out the scattered radiation has observable ramifications only in the EUV band
https://arxiv.org/pdf/2303.07412

PUEO is also one of the earliest to come (2025) UEHCR next gen telescope

As for GIGABIT, it will be a slighly larger telescope than euclid and will have an even more wide field camera, meaning we are entering the era where the best telescopes in the world are not space based but balloon based (not because of technicalities but because of widespread fraud)
and btw it is more complete since it is the first UVOIR since Hubble!
https://arxiv.org/abs/2407.10103
At the end of the year ASTHROS will be launched with a 2.5 meter aperture!! but since researchers all have ADHD they decided to make it far infrared, which is the biggest missed opportunity ever to obscolete euclid, Roman and Xuntian simultaneously while being launched 2 years earlier a 1/1000 of the cost.

**Comment by LifeIsStrange** [comment]
About balloons, there are three main launch sites:
Antarctica
New Zealand
And Sweden
Recently 4 balloons have been simultaneously launched and have completed their flight via Sweden.
For some strange reason the arctic route seems to be expected to be short (6 days of flight time while Antarctica can aims for 40 days of flight time (when the weather is right)
Anyway I never heard of those NASA sponsored balloons before yet they seems highly ambitious
https://www.nasa.gov/centers-and-facilities/wallops/nasa-balloons-head-north-of-arctic-circle-for-long-duration-flights/
HELIX (High-Energy Light Isotope eXperiment): A balloon-borne experiment that features a powerful superconducting magnet designed to measure the flux of high-energy cosmic ray isotopes to energies that have not been explored. The measurements will help determine the age of cosmic rays in our galaxy.
[BOOMS (Balloon Observation of Microburst Scales)](https://www.nasa.gov/general/lofted-by-nasa-balloons-new-experiments-will-study-sun-earth-system/#hds-sidebar-nav-4): A high-resolution imager of X-rays from energetic electron microbursts that appear in the polar atmosphere. The mission will fly on a 60 million cubic feet balloon, a test flight set to qualify the balloon for reaching altitudes greater than 150,000 feet, which is higher than NASA’s current stratospheric inventory.
SUNRISE-III: A solar observatory that takes high-resolution imaging and spectro-polarimetry of layers of the Sun called the solar photosphere and chromosphere, and active regions to measure magnetic field, temperature, and velocities with high height temporal resolution.
XL-Calibur: A telescope that will observe a sample of galactic black hole and neutron star sources to gain new insight on how these objects accelerate electrons and emit X-rays.

About them, HELIX might have just made the most accurate measurement of secondary cosmic rays, which should allow to confirm the existing anomaly for new physics.

Sunrise III might allow the most accurate measurements of the Sun (though of unclear competitivity with Dkis, the Solar orbiter and Parker.

GUSTO has also been recently completed and broke the duration record
https://ui.adsabs.harvard.edu/abs/2019AGUFM.A33Q2940B/abstract
It falls under the Missions of opportunity category 
Of the explorers program 

As for space telescope proposals, SALTUS stands out as it has a 14 meter aperture! Via an inflatable mirror, this concept is absolutely mind blowing and might allow for even larger apertures
https://arxiv.org/abs/2405.12829

https://www.centauri-dreams.org/2021/03/04/a-method-for-creating-enormous-space-telescopes/
> A demonstration EST with an aperture larger than the Webb Space Telescope could be flown on a CubeSat mission in low Earth orbit. ESTs with apertures of hundreds of meters could be delivered to heliocentric space in single flights of existing launch vehicles.
This little demonstration EST, with a total mass less than 20 kg, including optics that would be positioned along or suspended from the tether at the parabola focal point, would have four times the light gathering capacity of Webb (about thirty times that of Hubble), while costing on the order of 1/1000th as much.
An even cheaper flight demonstration could be done suspending an EST from a high altitude balloon
we find that a practical size limit for relatively near-term EST systems would be an aperture diameter of about 1000 meters. The discoveries that might be enabled by such systems are beyond reckoning.

Another futuristic breakthrough would be to leverage the solar gravitational lens
https://en.m.wikipedia.org/wiki/Solar_gravitational_lens

There is also the prospect of liquid mirrors 
https://en.m.wikipedia.org/wiki/Liquid-mirror_space_telescope
Again liquid mirrors are very much feasible, the first 4 meter telescope was recently built 
https://arxiv.org/abs/2311.05623
Before it there was the 6 meter zenith telescope, built for under a million which is absolutely revolutionary. 
It appears there might be no reason why mankind hasn't built a 50meters zenith telescope besides ineptitude. 
https://en.m.wikipedia.org/wiki/Large_Zenith_Telescope

https://arxivblog.com/?p=525
Note that their results on earth will be disappointing because they don't have adaptative optics though it could in theory have the best adaptative optics via software controlled ferromagnets but this hasn't yet been made a reality. Most liquid telescopes also cannot point but this isn't a huge limitation for cosmology or transient astronomy. Hence liquid telescopes in space or in the moon craters are highly promising.
The University plans a larger 8-meter liquid-mirror telescope named the Advanced Liquid-Mirror Probe (ALPACA) for astronomical use at an estimated first-light cost of $5 million, $3 million contingency, $10 million for the camera, $5 million for a [spectrograph](https://en.m.wikipedia.org/wiki/Optical_spectrometer), and $0.3 million operating costs per year.[[5]](https://en.m.wikipedia.org/wiki/Large_Zenith_Telescope#cite_note-5)
In April 2022, [NASA](https://en.m.wikipedia.org/wiki/NASA) reported that they would conduct the Fluidic Telescope Experiment (FLUTE) in the [ISS](https://en.m.wikipedia.org/wiki/ISS), which would be part of the [Axiom Mission 1](https://en.m.wikipedia.org/wiki/Axiom_Mission_1) astronaut [Eytan Stibbe](https://en.m.wikipedia.org/wiki/Eytan_Stibbe)'s research portfolio. The research would test liquid lens by using water injected by [polymers](https://en.m.wikipedia.org/wiki/Polymer) in [microgravity](https://en.m.wikipedia.org/wiki/Microgravity) through utilizing [buoyancy](https://en.m.wikipedia.org/wiki/Buoyancy) to even gravitational forces and cause [weightlessness](https://en.m.wikipedia.org/wiki/Weightlessness), to be later hardened by [UV light](https://en.m.wikipedia.org/wiki/UV_light) or [temperature](https://en.m.wikipedia.org/wiki/Temperature) in-orbit.[[2]](https://en.m.wikipedia.org/wiki/Liquid-mirror_space_telescope#cite_note-2)

A larger project is planned, called LAMA, with 66 individual 6.15-meter telescopes with a total collecting power equal to a 55-meter telescope, resolving power of a 70-meter scope.[[6]](https://en.m.wikipedia.org/wiki/Large_Zenith_Telescope#cite_note-6)[[7]](https://en.m.wikipedia.org/wiki/Large_Zenith_Telescope#cite_note-7)
The nasa project proposal is 50meters!!!
And has been selected for phase II
May 2024: FLUTE was [selected](https://www.nasa.gov/directorates/stmd/nasa-doubles-down-advances-six-innovative-tech-concepts-to-new-phase/) by NASA’s Innovative Advanced Concepts (NIAC) program for a Phase II study.

MY MIND IS SO BLOWN RIGHT NOW 
THE INFLATABLE MIRROR CONCEPT IS NOT JUST A VAPORWARE PIPE DREAM, 
It has already been launched to space and inflated successfully in 1996!!!!!! Making it the largest space radio telescope ever with a 14 meter aperture and they are so ineplty fraudulent this "test" wasn't filled with a dumb radio receiver making it an instrumentless mirror........ 
https://en.m.wikipedia.org/wiki/Inflatable_Antenna_Experiment
The history is reviewed in this paper:
https://arxiv.org/abs/2405.18529


https://spaceref.com/status-report/the-expanded-giant-metrewave-radio-telescope/

Long durAtion evalUation solaR hand LAunch (LAURA): 
Shows the need for diesel or nuclear battery powered balloons
https://www.nasa.gov/missions/scientific-balloons/nasa-scientific-balloons-ready-for-flights-over-antarctica/

https://www.nasa.gov/image-article/on-cupid-on-donner-and-barrel/

More on new ambitious large telescopes technologies
https://arxiv.org/abs/2109.12718
https://www.semanticscholar.org/paper/272f717214acb68aa4935dbebf77a09871aed057
https://www.semanticscholar.org/paper/9958f15b08a12dcffec6a657ef80ce35d6e36c62
https://www.centauri-dreams.org/2019/08/13/the-terrascope-challenges-going-forward/

More on liquid telescopes
https://www.nature.com/articles/s41526-023-00309-9

Another paradigm is to use foldable membrane mirrors
https://www.optica.org/about/newsroom/news_releases/2023/april/researchers_devise_new_membrane_mirrors_for_large/

There are also diffractive lenses like NAUTILUS with aperture synthesis.

Another direction and the most obvious one to do larger telescopes is to make larger segmented mirrors like the JWST and LUVOIR.
Segmented mirrors should be cheap and abundant but there isn't a single project in the 2020s to bring a segmented mirror.... (except a small one on a cubesat)
Some ideas can be combined like segmented foldable mirrors.
Obviously the main way to the future telescope is to do a dumb 6.5 meters monolith and the fact it hasn't yet been done despite the ridiculously low price is a crime against humanity.

Monolithic glass mirrors are limited to ~11meters
Hence it is interesting to consider alternative materials that might be cheaper and more scalable.
The main contender of Esa, is silicon carbide.
Recently for the first time two 4 meters mirrors were made, showing that sic is not yet competitive.
The most forgotten alternative is to use metals, as was the case for the first telescopes (speculum)
There are some issues with most metals,
1) oxidation however this does not happen in space!
2) thermal expansion

The two main contenders are beryllium and aluminum.
Beryllium is used on JWST and is 
 expensive. However aluminum might be the future of large and cheap space telescopes
The Esa for the first time is developing one for Ariel
https://www.esa.int/Enabling_Support/Space_Engineering_Technology/To_see_the_Universe_in_aluminium
Sadly while successful, they haven't tried to scale it to a large size...
There is also an enterprise but no size published
https://www.coherent.com/news/blog/aluminum-mirrors-space-telescopes
So it's very promising but not sure wether anybody will fund it.

There is also the topic of bypassing the diffraction limit via negative lenses or other technologies though this is currently more being developed for microscopes
https://www.sciencedirect.com/science/article/pii/S2095809924001498?via%3Dihub
https://arxiv.org/pdf/2107.00664

I can't believe I missed new chinese space telescopes!
https://cosparhq.cnes.fr/assets/uploads/2020/12/China_CAS_2020.pdf
there is also this twitter journalist specialized on the topic
https://twitter.com/AJ_FI
Miyin is a planned (prototype launch 2027, final 2030) optical interferometer for exoplanet search probably similar to LIFE interferometer
https://spacenews.com/china-to-hunt-for-earth-like-planets-with-formation-flying-telescopes/
and even more impressive, is the Earth 2.0 (ET) mission that should launch in 2026!
https://arxiv.org/abs/2206.06693
there is also Selected missions include Discovering the Sky at the Longest Wavelength (DSL), the Enhanced X-ray Timing and Polarimetry (eXTP) mission, and the ExoEarth Survey
https://english.www.gov.cn/archive/whitepaper/202201/28/content_WS61f35b3dc6d09c94e48a467a.html
http://english.nssc.cas.cn/news/202407/t20240712_672961.html

aside from alluminium, there is an alternative mirror material that is cheap, scalable and that already exists: borosillicate glass
https://www.researchgate.net/publication/252251703_Low-weight_Low-cost_Low-cycle_Time_Replicated_Glass_Mirrors
It is used on many large telescopes including the incoming GMT.
An interesting 6.5 meters ground based one is 
https://photonix.springeropen.com/articles/10.1186/s43074-023-00094-4
which aims to have 19 times the bandwidth of DESI

For exoplanets there is also the exoworlds project of ISRO which aims for a launch between 2025 and 2028
It apparently aims to be the most ambitious exoplanet mission ever though I have huge doubts about it considering the very limited budget of ISRO
https://forum.nasaspaceflight.com/index.php?topic=56822.0

There is also the prospect of very large balloon telescopes
A 20–30 meter balloon telescope has been suggested.[[31]](https://en.m.wikipedia.org/wiki/List_of_proposed_space_telescopes#cite_note-auto-31) The balloon would be transparent on one side, and have a circular reflecting mirror on the other side.[[31]](https://en.m.wikipedia.org/wiki/List_of_proposed_space_telescopes#cite_note-auto-31) There are two main designs using this principle.[[31]](https://en.m.wikipedia.org/wiki/List_of_proposed_space_telescopes#cite_note-auto-31)

Large Balloon Reflector (LBR) (sub-orbital version)
Space-based Large Balloon Reflector (LBR)
TeraHertz Space Telescope (TST)[[32]](https://en.m.wikipedia.org/wiki/List_of_proposed_space_telescopes#cite_note-32)
LAunch readiness 2024 for the TST yet it is non funded....
The project seemed to be dead but they did some test launches in 2023
https://news.ycombinator.com/item?id=38043955
https://www.freefallaerospace.com/nasa-balloon/
Catsat launched two weeks ago 
https://youtu.be/7ujMP8Ao4bI

Note xuntian might do some thz spectroscopy although that is unclear
https://astronomy.stackexchange.com/questions/53065/how-will-chinas-xuntian-space-telescope-use-its-terahertz-capability-thz-spect

While the Pandora mission isn't interesting, It is the first space mission to deploy an aluminium mirror
https://arxiv.org/abs/2108.06438
More interesting smallsats being CDEEP, aspera and stamp-1
https://astrobiology.com/2024/07/the-smallsat-technology-accelerated-maturation-platform-1-stamp-1-a-proposal-to-advance-ultraviolet-science-workforce-and-technology-for-the-habitable-worlds-observatory.html

There are also some interesting Russian space missions projects, like the sepktr M deployable antenna
https://www.russianspaceweb.com/spektr_m.html
https://www.russianspaceweb.com/spacecraft_science.html

Also https://arxiv.org/abs/2304.06482v5
https://www.researchgate.net/publication/382462433_Development_of_the_High-resolution_Spectrometer_of_the_Millimetron_Space_Observatory


https://www.eoportal.org/satellite-missions/falconsat-7#falconsat-7
See figure 1

another genius technology to allow for large space telescopes, is to make a mirror highly rectangular, which is the shape of a space rocket and allow for very lengthy mirrrors.
Then the rectangular mirror is continuously rotated, effectivelly forming a perfect sphere!
Through algorithms, it is possible to perform aperture synthesis considerably better than regular sparse arrays!!
https://www.researchgate.net/publication/319497956_Revolutionary_astrophysics_using_an_incoherent_synthetic_optical_aperture
https://news.ycombinator.com/item?id=41076063

Damn the new FAST wideband receiver allows FAST to outperform SKA mid, this is crazy
https://arxiv.org/abs/2408.00268

https://arxiv.org/abs/2402.00561

X-ray characterization of exoplanets
https://arxiv.org/abs/2408.06417
